### DISCUSSION TITLE
Items of different size

### DISCUSSION THREAD

**Discussion by inzanez** [rocwang/vue-virtual-scroll-grid#71] [Q&A] [answered]
Hi

is there a way to have items of different size being displayed? I imagine something like a 'portrait' and 'landscape' photo,...or does `vue-virtual-scroll-grid` always needs the same size for every item?

**Comment by rocwang** [comment, marked as answer]
Yes, this is a known limitation.

> ## Caveats
> The library does not require items have foreknown width and height, but do require them to be styled with the same width and height under a view. E.g. the items can be 200px x 200px when the view is under 768px and 300px x 500px above 768px.

I couldn't find a quick solution to support items with various sizes. One can wrap the item in a fixed sized frame element, but that might not be what you want as there would be gaps between items.
